src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -43,15 +51,34 @@ const AppealsV2StatusPage = ({ appeal, fullName }) => {
     !hideDocketStatusTypes.includes(status.type) &&
     !hideDocketAppealTypes.includes(appealType);
 
+  const filteredAlerts = alerts.filter(a => a.type !== ALERT_TYPES.cavcOption);
+  const afterNextAlerts = (
+    <div>
+      {alerts
+        .filter(a => a.type === ALERT_TYPES.cavcOption)
+        .map((a, i) => {
+          const alert = getAlertContent(a, appealIsActive);
+          return (
+            <div key={`after-next-alert-${i}`}>
+              <h2>{alert.title}</h2>
+              <div>{alert.description}</div>
+            </div>
+          );
+        })
+      }
+    </div>
+  );
+
   return (
     <div>
       <Timeline events={events} missingEvents={incompleteHistory}/>
       <CurrentStatus
         title={currentStatus.title}
         description={currentStatus.description}
         isClosed={!appealIsActive}/>
-      <AlertsList alerts={alerts} appealIsActive/>
+      <AlertsList alerts={filteredAlerts} appealIsActive/>
       {appealIsActive && <WhatsNext nextEvents={nextEvents}/>}
+      {afterNextAlerts}
       {shouldShowDocket && <Docket {...docket} aod={aod} form9Date={form9Date} appealType={appealType}/>}
```

Comment:
This change was not made here, but would it be of interest to use destructuring consistently here?

Reply:
How do you mean?

Reply:
Never mind, I misread where these props are coming from, apologies! Looks good!